comp_conf/RRFS-CMAQ-DA/build_scripts/build_GSI.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -eu
+
+if [[ $(uname -s) == Darwin ]]; then
+  readonly MYDIR=$(cd "$(dirname "$(greadlink -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
+else
+  readonly MYDIR=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
+fi
+
+SRW_APP_DIR="${MYDIR}/../../.."
+COMP_DIR="${SRW_APP_DIR}/comp_conf"
+SRC_DIR="${SRW_APP_DIR}/src/gsi"
+BUILD_DIR="${SRW_APP_DIR}/build/gsi"
+BIN_DIR="${SRW_APP_DIR}/bin/"
+
+# Detect MACHINE
+source ${COMP_DIR}/detect_machine.sh
+
+echo "MACHINE:" ${MACHINE}
```

Comment:
This GSI is not the same as the authoritative one. It was modified only for RRFS-CMAQ. The developer will merge his change into the authoritative repo in the future. Until then, this GSI should be separated from the authoritative one. Currently, RRFS-CMAQ works only on Hera and WCOSS Dell, and RRFS-CMAQ-DA works only on Hera because JEDI is only available on Hera now.